sdk/storage/azure-storage-blob-stress/templates/stress-test-job.yaml

```diff
@@ -23,6 +27,7 @@ spec:
           export APPLICATIONINSIGHTS_ROLE_NAME=storage-{{ .Stress.BaseName }};
           java \
           -javaagent:BOOT-INF/classes/applicationinsights-agent-3.4.17.jar \
+          -Xms1g -Xmx1500m \
```

Comment:
TODO: to check if it should have any effect

Reply:
hard to say, based on the [oracle's docs](https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/memleaks002.html)

> One common indication of a memory leak is the java.lang.OutOfMemoryError exception. Usually, this error is thrown when there is insufficient space to allocate an object in the Java heap. In this case, The garbage collector cannot make space available to accommodate a new object, and the heap cannot be expanded further. Also, this error may be thrown when there is insufficient native memory to support the loading of a Java class. In a rare instance, a java.lang.OutOfMemoryError may be thrown when an excessive amount of time is being spent doing garbage collection and little memory is being freed.

but let's keep xms high for the time being.